Fix and enhance troubleshooting (#4377)
harness/developer-hub: docs/platform/delegates/troubleshooting/certificate-issues.md

   curl -cacerts path/to/ca-certs/file https://<MANAGER_HOST>/api/account/<ACCOUNT_ID>/status
   ```

> 2. Install the certificate on the delegate. go to [Install delegates with custom certificates](/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs/).

Reviewer: ```suggestion
2. Install the certificate on the delegate. For more information, go to [Install delegates with custom certificates](/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs/).
```

         
   3. Copy the `BEGIN CERTIFICATE` and `END CERTIFICATE` blocks into a new `cacerts.pem` file.

>    4. Add the CA certificates to the delegate. go to [Install delegates with custom certificates](/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs/).

Reviewer: ```suggestion
   4. Add the CA certificates to the delegate. For more information, go to [Install delegates with custom certificates](/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs/).
```

    
    3. If the installation succeeds, following the OpenSSL steps above. If the installation fails, continue with the steps below.

> 2. Use the cURL commands below to find the issuers that are missing in your CA bundle. Find the certificate for each issuer by going to the domain in your browser or download the certificate.

Reviewer: ```suggestion
2. Use the cURL commands below to find the issuers that are missing in your CA bundle. Find the certificate for each issuer by going to the domain in your browser and downloading the certificate.
```

   
   ```
   curl -vk <YOUR_URL>
   ```

> 3. Turn on SSL debug log. Set JAVA_OPTS environment variable when installing delegate

Reviewer: ```suggestion
3. Turn on the SSL debug log by setting the `JAVA_OPTS` environment variable when installing delegate.

```
